scripts/ci/get_image_tags_from_branch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -xe
+GIT_BRANCH=$1
+IMAGE_VERSION=$2
+BUILD_NUMBER=$3
+commit_sha=$4
+branch_tag=`echo $GIT_BRANCH| sed 's|/|.|g'`  #not sure if docker accept / in the version
```

Comment:
I don't understand the comment. Can you please explain?

Reply:
[deja vu](https://github.com/IBM/ibm-block-csi-driver/pull/344#discussion_r676112231)

Reply:
this was a comment that was there before, I don't know if we really need it, I just didn't want to remove it

Reply:
please dedup existing open conversations